### PR TITLE
Issue #354 - Add article & reply time

### PR DESCRIPTION
**Pull request by jtsaich**
For Issue #354 
Screenshot 2024-01-06 at 2 27 10 PM
Screenshot 2024-01-06 at 2 57 10 PM
Screenshot 2024-01-06 at 3 01 04 PM

Localization is less than perfect but the date should be displaying correctly

### CODE REVIEW

File: src/graphql/__tests__/userArticleLinks.js
Comment (MrOrz) [inline src/graphql/__tests__/userArticleLinks.js:1]: This looks like a flaky test result -- the `userArticleLinks` becomes empty for unknown reason. Please help revert the change of this particular file.

File: src/scripts/__tests__/__snapshots__/lib.js.snap
Comment (MrOrz) [inline src/scripts/__tests__/__snapshots__/lib.js.snap:1]: This also seems like a flaky test result, please help revert the change of this particular file 🙏🏻

**Review by MrOrz — COMMENTED** [review]
Thanks for the update! Here are some quick comments.

**Comment by jtsaich** [comment]
@MrOrz 有些 snapshot test 是不是會一直變呀

**Comment by MrOrz** [comment]
> @MrOrz 有些 snapshot test 是不是會一直變呀

是的，再麻煩 commit 時忽視這些檔案 QQ

File: src/webhook/handlers/__tests__/__snapshots__/askingArticleSubmissionConsent.test.ts.snap
Comment (MrOrz) [inline src/webhook/handlers/__tests__/__snapshots__/askingArticleSubmissionConsent.test.ts.snap:1]: The changes in this file also looks like a flaky result @@

File: src/webhook/handlers/choosingArticle.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ import choosingReply from './choosingReply';
 import type { Input as ChoosingReplyInput } from './choosingReply';
 import { ChatbotPostbackHandler } from 'src/types/chatbotState';
 import { FlexBubble, Message } from '@line/bot-sdk';
+import { addDays, formatDistance, isBefore } from 'date-fns';
```
Comment (MrOrz) [inline src/webhook/handlers/choosingArticle.ts:33]: nit: remove unused import here

File: src/webhook/handlers/__tests__/__snapshots__/singleUserHandler.test.ts.snap
Comment (MrOrz) [inline src/webhook/handlers/__tests__/__snapshots__/singleUserHandler.test.ts.snap:1]: This is flaky result too @@

**Review by MrOrz — COMMENTED** [review]
Thanks again for the change. I don't even remember we have `format()` in `sharedUtils`; you must have digged deep into the code.

I think we are good to merge after the remaining flaky test snapshots are removed and the wording issue being fixed :+1:

**Comment by johnson-liang** [comment]
No worries, I will take over so that these can be tested tomorrow

**Pull request merged**